xml/templates/message_templates/event_offline_receipt_html.tpl

```diff
@@ -28,12 +28,12 @@
 
     {if !empty($isOnWaitlist)}
      <p>{ts}You have been added to the WAIT LIST for this event.{/ts}</p>
-     {if !empty($isPrimary)}
+     {if {event.is_monetary|boolean}}
```

Comment:
I'm not familiar with how the wait list system works, but do participants get registered automatically if space becomes available in the event if the event is non-monetary?

Reply:
@larssandergreen yeah I think that must be the case

Reply:
I checked the docs and tested. You do get an email with a link that you have to click to move from waitlist to registered, even for a free event. I think this was probably, as you say, copied from the online template, where it was supposed to prevent additional participants from getting emails (it only emails the primary participant to confirm for all participants in the group). Same for approval. So I think we can remove both these conditionals.

Reply:
Thanks for the investigation - I have pushed up a fix to remove them! Yay for simplification

Reply:
Any message template simplification is a happy occasion.

Reply:
You are so right @larssandergreen